### PR TITLE
Fixes 868 - Delete sub sub sub albums bug

### LINKED ISSUE #868: Deleting double-nested albums fails
```
+----------------+----------------+-------+------+------+
| id             | parent_id      | title | _lft | _rgt |
+----------------+----------------+-------+------+------+
| 16106622698316 |           NULL | 1     |   77 |   82 |
| 16106622758220 | 16106622698316 | 2     |   78 |   81 |
| 16106622793387 | 16106622758220 | 3     |   79 |   80 |
+----------------+----------------+-------+------+------+
```

When I try to delete `1`, I get a server error, with the following in the logs:

```
[2021-01-14 16:12:53] local.INFO: select `photos`.* from `photos` left join `albums` on `photos`.`album_id` = `albums`.`id` where `albums`.`_lft` >= ? and `albums`.`_rgt` <= ? [77,82] 
[2021-01-14 16:12:53] local.ERROR: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`homestead2`.`albums`, CONSTRAINT `albums_parent_id_foreign` FOREIGN KEY (`parent_id`) REFERENCES `albums` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT) (SQL: delete from `albums` where `_lft` > 77 and `_rgt` <= 82) {"exception":"[object] (Illuminate\\Database\\QueryException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`homestead2`.`albums`, CONSTRAINT `albums_parent_id_foreign` FOREIGN KEY (`parent_id`) REFERENCES `albums` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT) (SQL: delete from `albums` where `_lft` > 77 and `_rgt` <= 82) at [...]/Lychee/vendor/laravel/framework/src/Illuminate/Database/Connection.php:678)
```

### PR DESCRIPTION
**Pull request by ildyria**
@kamil4 here is your fix

TL;DR: we temporarily disable the _ForeignKeyConstraints_.

Fixes #868

### CODE REVIEW

**Review by kamil4 — APPROVED** [review]
Well, my nested album now deletes fine :smiley:.
